src/commands/run.ts

```diff
@@ -18,7 +18,7 @@ interface RunArgv extends CommonArgv {
   rootDatabase?: boolean;
 }
 
-export async function run(
+export async function run<T = void>(
```

Comment:
`void` is definitely wrong here, `never` would be a more appropriate type but I think `<T = any>` is preferred.
```suggestion
export async function run<T = any>(
```

Reply:
So, I got this by following that we return the call to pgReal.withClient, which is `<T = void>` and returns a `Promise<T>`? Happy to make it an any (although es-lint will then complain) if you're sure. To get around the complaining, can I just eslint-ignore that line still? is that still even a thing?